model/histogram/generic.go

```diff
@@ -58,6 +58,12 @@ func IsValidSchema(s int32) bool {
 	return IsCustomBucketsSchema(s) || IsExponentialSchema(s)
 }
 
+// IsAcceptibleSchema returns bool if we accept the schema, but need to reduce
+// resolution to the nearest supported schema.
+func IsAcceptibleSchema(s int32) bool {
```

Comment:
```suggestion
func IsAcceptableSchema(s int32) bool {
```

"Acceptible" is quite outdated.

Reply:
I've renamed this to IsUnknownSchema which is easier to understand and spell :)